roles/confluent.test/molecule/plain-rhel/molecule.yml

```diff
@@ -100,6 +100,20 @@ provisioner:
       all:
         sasl_protocol: plain
 
+        # Testing secrets protection
+        secrets_protection_enabled: true
+        confluent_cli_download_enabled: true
+
+        # Test secrets protection in kafka broker
+        kafka_broker_secrets_protection_config_path: "./etc/kafka/server.properties"
+        kafka_broker_secrets_protection_config_params: [advertised.listeners, broker.id]
```

Comment:
is there a better word than params? makes me think of parameters. what do the docs call the secrets properties

Reply:
In the docs, they call them "configuration parameters" so that's why I went with params